contracts/utility/NFTForwarding.cdc

```diff
@@ -2,46 +2,47 @@
 ///
 /// This contract enables a user to designate a recipient so NFTs could be forwarded
 ///
-/// The NFTForwarder resource can be referenced just like any NonFungibleToken Receiver,
+/// The NFTForwarder resource can be referenced just like any NonFungibleToken Collection,
 /// allowing a sender to deposit NFTs as they usually would
 ///
 /// However, in this implementation, any time a deposit is made, the deposited NFT is
-/// additionally deposited to a predefined recipient.
+/// additionally deposited to a predefined recipient Collection.
+///
+/// To create an NFTForwarder resource, an account calls the createNewNFTForwarder()
+/// function, passing the Collection Capability to which NFTs will be forwarded.
 ///
-/// To create an NFTForwarder resource, an account calls the createNewNFTForwarder
-/// function, passing the Receiver Capability to which NFTs will be forwarded.
-
 import NonFungibleToken from "NonFungibleToken"
 
-pub contract NFTForwarding {
+access(all) contract NFTForwarding {
+
+    access(all) entitlement Mutable
 
-    pub event ForwardedNFTDeposit(id: UInt64, from: Address?)
-    pub event NFTForwarderRecipientChanged(forwarder: Address?)
+    access(all) event ForwardedNFTDeposit(id: UInt64, from: Address?)
```

Comment:
Can this event contain type and UUID fields like NFT have added now?

Reply:
yep, I'll add